members/actions.py

```diff
@@ -0,0 +1,75 @@
+#coding: utf-8
+
+import unicodecsv
+from .membership import InvoiceTemplate
+from django.http import HttpResponse
+from django.conf import settings
+from django.utils.encoding import smart_str
+import os
+
+
+def export_as_csv_action(description="Izvoz odabranih stavki u CSV",
+                         fields=None, exclude=None, header=True):
+    """
+    This function returns an export csv action
+    'fields' and 'exclude' work like in django ModelForm
+    'header' is whether or not to output the column names as the first row
+    """
+    def export_as_csv(modeladmin, request, queryset):
+
+        opts = modeladmin.model._meta
+
+        if not fields:
+            field_names = [field.name for field in opts.fields]
+        else:
+            field_names = fields
+
+        response = HttpResponse(mimetype='text/csv')
+        response['Content-Disposition'] = 'attachment; filename=%s.csv' % unicode(opts).replace('.', '_')
+
+        writer = unicodecsv.writer(response, encoding='utf-8')
+        if header:
+            writer.writerow(field_names)
+        for obj in queryset:
+            row = [__get_field(obj, field)() if callable(__get_field(obj, field)) else __get_field(obj, field) for field in field_names]
+            writer.writerow(row)
+        return response
+    export_as_csv.short_description = description
+    return export_as_csv
+
+
+def make_bill(modeladmin, request, queryset):
```

Comment:
This isn't mine code, I don't this ever worked. Just moved this in separe file with my new action...